UI/src/components/widgets/score/config.js

```diff
@@ -0,0 +1,107 @@
+(function () {
+    'use strict';
+
+    angular
+        .module(HygieiaConfig.module)
+        .controller('scoreConfigController', scoreConfigController);
+
+    scoreConfigController.$inject = ['modalData', 'collectorData','$uibModalInstance'];
+    function scoreConfigController(modalData, collectorData, $uibModalInstance) {
+        /*jshint validthis:true */
+        var ctrl = this;
+
+        var widgetConfig = modalData.widgetConfig;
+
+        // public variables
+        ctrl.submitted = false;
+        ctrl.collectors = [];
+        ctrl.collectorItemId = null;
+        ctrl.settingsType = 'DEFAULT';
```

Comment:
@sidbha007  - are there other settingTypes ?

Reply:
We used to have settings type as DEFAULT and CUSTOM for dashboard. Since we are not using CUSTOM and everything is DEFAULT, have removed the property for settingsType